server/src/Service/InteractionService.php

```diff
@@ -135,4 +142,31 @@ private function validateUsers( array $users ) {
 			throw new \InvalidArgumentException( 'at least 2 users should be provided' );
 		}
 	}
+
+	/**
+	 * Takes a timestamp or a string representing a date
+	 * and returns a \DateTime object
+	 *
+	 * @param int|string $date
+	 * @return \DateTime|null
+	 * @throws \InvalidArgumentException
+	 */
+	private function sanitizeDate( $date ) {
+		if ( $date ) {
+			if ( is_numeric( $date ) ) {
+				$dateObj = new \DateTime();
+				$dateObj->setTimestamp( $date );
```

Comment:
Use https://secure.php.net/manual/en/datetime.createfromformat.php with `U` as the format. :)

Reply:
Wow, I swear I looked for something like that and didn't find it. Thanks. Changing it